### PR TITLE
Atmospheric, Speed setting, new plasma reaction, Fire does less damage

### LINKED ISSUE #2438: Large fires no longer happen?
# Description
Fires no longer expand regardless of plasma/oxygen amounts in the room.
Reproducible locally.

## Steps to Reproduce
Please enter the steps to reproduce the bug or behaviour:
1. Spawn in atmosia
2. Open plasma and oxygen tanks
3. Turn on a welder or artificially hotspot some item via right click menu
4. Observe a small fire that dies down in a second, while it should spread like hell and take over the station

### PR DESCRIPTION
**Pull request by Bod9001**
Co-Authored-By: Arvenius <arvenius169@users.noreply.github.com>
it's mainly all his work

### Purpose
cleaned out unneeded verbal
renames to proper names 
adds a small bit of documentation
Fixes #2438
by implementing a new fire reaction The main issue was that it was being quickly dissipated over the adjacent Tiles to for it to work properly with the old fire reaction
Adjusted speed of the atmospheric system to ensure fire doesn't heat up every part of a room No matter how big
Changed damage from fire make it more like the byond Version

### CODE REVIEW

**Comment by Bod9001** [comment]
Still working on this a bit

File: UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/AtmosManager.cs [hunk 1]
```diff
@@ -6,8 +6,7 @@
 
 public class AtmosManager : MonoBehaviour
 {
-	public float Speed = 0.01f;
-
+	public float Speed = 40;
```
Comment (chairbender) [inline UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/AtmosManager.cs:12]: Units plz. I think this is milliseconds / tick ?

**Review by chairbender — APPROVED** [review]
Nicely done arvenius, that's quite the first time PR. And thanks Bod for putting this together.

Just wanted units documented on that one variable.

Looks good to me. Fires work better now, even some other things seem to be working better. I approve, but I'm going to leave this open so I can get at least one additional opinion
(@DooblyNoobly @Jibbajabbafic @fomalsd )

File: UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/AtmosManager.cs (hunk 1 above)
Comment (Bod9001) [reply UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/AtmosManager.cs:12]: yeah is milliseconds

File: UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/Data/Reaction.cs [hunk 2]
```diff
@@ -24,44 +38,56 @@ public float React(ref GasMix gasMix)
...
-			float temperatureScale = 1;
-			if (temperature <= Reactions.PLASMA_UPPER_TEMPERATURE)
-			{
-				temperatureScale = (temperature - Reactions.PLASMA_MINIMUM_BURN_TEMPERATURE) /
-				                   (Reactions.PLASMA_UPPER_TEMPERATURE - Reactions.PLASMA_MINIMUM_BURN_TEMPERATURE);
+			float BurnRate = GetOxygenContact(gasMix);
+			//Logger.Log(BurnRate.ToString() + "BurnRate");
+			if (BurnRate > 0)
+			{
+				float moles = gasMix.GetMoles(Gas.Plasma) * Reactions.BurningDelta * BurnRate;
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/Data/Reaction.cs]: raname

**Comment by fomalsd** [comment]
I suck at physics, but code looks good (yet to playtest)
Is this ready to merge or are you planning to work on it some more?

File: UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/AtmosManager.cs (hunk 1 above)
Comment (chairbender) [reply UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/AtmosManager.cs:12]: please add the units as a comment

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/AtmosManager.cs:12]: ok

**Comment by Bod9001** [comment]
Just a few renams

**Comment by Bod9001** [comment]
is good now

**Pull request merged**